modules/olm-accessing-images-private-registries.adoc

```diff
@@ -0,0 +1,173 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/managing-custom-catalogs.adoc
+
+[id="olm-accessing-images-private-registries_{context}"]
+= Accessing images for Operators from private registries
+
+If certain images relevant to Operators managed by Operator Lifecycle Manager (OLM) are hosted in an authenticated container image registry, also known as a private registry, OLM and OperatorHub are unable to pull the images by default. To enable access, you can create a pull secret that contains the authentication credentials for the registry. By referencing one or more pull secrets in a catalog source, OLM can handle placing the secrets in the Operator and catalog namespace to allow installation.
+
+Other images required by an Operator or its Operands might require access to private registries as well. OLM does not handle placing the secrets in target tenant namespaces for this scenario, but authentication credentials can be added to the global cluster pull secret or individual namespace service accounts to enable the required access.
+
+The following types of images should be considered when determining whether Operators managed by OLM have appropriate pull access:
+
+Index or catalog images:: Index images and catalog images are containerized snapshots of an entire catalog. Index images reference bundle images. If an index or catalog image is hosted in a private registry, OperatorHub can use a secret to enable pull access.
+
+Bundle images:: Operator bundle images are containerized Kubernetes manifests and Operator metadata that represent a unique version of an Operator. If any bundle images referenced in a catalog are hosted in one or more private registries, OperatorHub can use one or more secrets to enable pull access.
+
+Operator and Operand images:: If an Operator subscribed from a catalog uses a private image, either for the Operator image itself or one of the Operand images it manages, the Operator will fail to install because the deployment will not have access to the required registry authentication. Referencing secrets in a catalog source does not enable OLM to place the secrets in target tenant namespaces.
```

Comment:
```suggestion
Operator and Operand images:: If an Operator installed from a CatalogSource uses a private image, either for the Operator image itself or one of the Operand images it watches, the Operator will fail to install because the deployment will not have access to the required registry authentication. Referencing secrets in a CatalogSource does not enable OLM to place the secrets in target tenant namespaces that operands are installed in.
```